schemas/event.yml

```diff
@@ -99,45 +220,108 @@
+        This is one of four ECS Categorization Fields, and indicates the
+        lowest level in the ECS category hierarchy.
 
-        If the event describes an action, this fields contains the outcome of
-        that action.
+        `event.outcome` simply denotes whether the event represent a success or a failure.
+        Note that not all events will have an associated outcome. For example, this field is
+        generally not populated for metric events or events with `event.type:info`.
       example: success
-
-      accepted_values:
-      - name: success
-        description: >
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
-          incididunt ut labore et dolore magna aliqua.
-
-      - name: failure
-        description: >
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
-          incididunt ut labore et dolore magna aliqua.
+      allowed_values:
+        - name: failure
+          description: >
+            Indicates that this event describes a failed result. A common example is
+            `event.category:file AND event.type:access AND event.outcome:failure`
+            to indicate that a file access was attempted, but was not successful.
+        - name: success
+          description: >
+            Indicates that this event describes a successful result.  A common example is
+            `event.category:file AND event.type:create AND event.outcome:success`
+            to indicate that a file was successfully created.
+        - name: unknown
+          description: >
+            Indicates that this event describes only an attempt for which the result
+            is unknown. For example, if the event contains information only about a
+            request in an entity transaction that usually results in a response,
+            populating `event.outcome:unknown` is appropriate.
 
     - name: type
       level: core
       type: keyword
-      short: Reserved for future usage.
+      short: Event type (Categorization Field)
       description: >
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit,
-        sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-
-      accepted_values:
-      - name: start
-        description: >
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
-          incididunt ut labore et dolore magna aliqua.
-
-      - name: end
-        description: >
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
-          incididunt ut labore et dolore magna aliqua.
+        This is one of four ECS Categorization Fields, and indicates the
+        third level in the ECS category hierarchy.
+
+        `event.type` represents a categorization "sub-bucket" that, when used along with
+        the `event.category` field values, enables filtering events down to a level
+        appropriate for single visualization.
```

Comment:
Yes, it enables visualization, but we should communicate also the value of this field beyond organization.
What does event type _mean_? How should I approach this field?